### ISSUE TITLE
Inconsistent K in Latin and Cyrillic

### ISSUE THREAD

**Issue by ghost** [google/roboto#289]
In Roboto 2015 latin letter K was updated, but the cyrillic one wasn't. They should be the same.
![image](https://user-images.githubusercontent.com/37714087/39092868-e8335018-461e-11e8-8101-717c2a6641b6.png)

**Comment by kushatc** [comment]
It should look like that:
![image](https://user-images.githubusercontent.com/37714087/39094795-2ebf9f4a-463e-11e8-8f5b-d81f825c37a6.png)

**Comment by bananakid** [comment]
This should be addressed as well as К-related Cyrillic letters like, i.e. Ж. Please release hinted version because release v2.138 never received hinting treatment.

**Comment by evlogii** [comment]
Why should they be the same? They are not the same in almost all fonts. And I believe they shouldn't cause they are different letters of different alphabets, and the more they differ, the better.

https://fonts.google.com/?preview.text=K%D0%9A&preview.text_type=custom

**Comment by fitojb** [comment]
Correct. This is not a bug. For this particular letter, it is expected, and advisable even, that a homoglyph is avoided by having different constructions in the Latin, the Greek and the Cyrillic renditions.
